Core/GDCore/Extensions/Builtin/TimeExtension.cpp

```diff
@@ -14,21 +14,26 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsTimeExtension(
   extension
       .SetExtensionInformation(
           "BuiltinTime",
-          _("Time"),
+          _("Timers and time"),
```

Comment:
It feels weird to have 2 times "time" in the text (pun intended)

What about `Time tools`? or `Time mechanics` or `Time manipulation`

Reply:
Agree but maybe let's not rename this in this PR, to avoid users having trouble finding this if they follow tutorials?
Also the repetition is not that much of a problem I would say, especially because it's different in other languages :)

Reply:
I'll keep this for now to avoid confusing people searching for this group :)